src/plugins/theme/fontFamily.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Font Family Defaults.
+ */
+module.exports = function () {
+  return {
+    mono: ['Roboto Mono', 'Menlo', 'Courier New', 'monospace'],
+    sans: ['Source Sans Pro', 'Helvetica Neue', 'Helvetica', 'Arial', 'sans-serif'],
+    serif: ['Source Serif Pro', 'Georgia', 'Times', 'Times New Roman', 'serif'],
+    slab: ['Roboto Slab', 'Georgia', 'Times', 'Times New Roman', 'serif'],
```

Comment:
No more handwriting or Sanskrit? For backwards compatibility, we may need to add them back in.

Reply:
Yeah this is the "ideal" version - without worrying about backward compatibility version we talked about :) Dana, Kerri and I think that we might be completely removing Kalam, and seems like the Sanskrit font doesn't belong in Decanter (more of a one-off thing).